README.md

```diff
@@ -66,6 +66,25 @@ Alternatively, you can build using provided Makefiles:
 make -f Makefile.manual
 ```
 
+## Memory issues during compilation
+
+Some issues can be encountered during compilation on low RAM computers.
+These issues are due to the high number of procedures generated by the preprocessor.
+They can be avoided by limiting the maximum rank of generated procedures, as follows with `CMake`:
+
+```sh
+cmake -B build
+
+cmake --build build -DCMAKE_MAXIMUM_RANK=4
```

Comment:
```suggestion
cmake -B build -DCMAKE_MAXIMUM_RANK=4

cmake --build build
```

Reply:
Indeed. I was too fast to copy my terminal... Thanks for the review @awvwgk .